server/wfnews-persistence/src/main/resources/ca/bc/gov/nrs/wfnews/persistence/v1/dao/mybatis/mapper/PublishedIncidentMapper.xml

```diff
@@ -333,6 +333,12 @@
 			<if test="fireCentreName != null &amp;&amp; fireCentreName != ''">  
 				AND fire_centre_name = #{fireCentreName, javaType=java.lang.String, jdbcType=VARCHAR, mode=IN}
 			</if>
+			<if test="discoveryDate != null">
+				AND DISCOVERY_DATE > #{discoveryDate, javaType=java.util.Date, jdbcType=DATE, mode=IN}
```

Comment:
You dont want these to be `>=` ?

Reply:
Active fire monitor insert the newest fire's ignition date(will be changed to insert createDate) as last_fetched_time_stamp, add = may get fires that were already reported during last fetch period.